### PR TITLE
docs: change the app name reference

### PR DESCRIPTION
**Pull request by geromegrignon**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The tutorial indicates : 
`An initial skeleton app project, also called angular-tour-of-heroes (in the src subfolder).`

The src subfolder of the skeleton app project is always named `app`, regardless of the name of the workspace

## What is the new behavior?

The next test is :
`An initial skeleton app project, called app (in the src subfolder).`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by geromegrignon** [comment]
the link of the failing legacy-units-tests-saucelabs isn't reachable so i'm not able to know what failed.

**Comment by AndrewKushnir** [comment]
@geromegrignon thanks for the PR! I've restarted the failing CI job, since it was a flake.

**Review by AndrewKushnir — APPROVED** [review]

File: aio/content/tutorial/toh-pt0.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The Angular CLI installs the necessary Angular `npm` packages and other dependen
 It also creates the following workspace and starter project files:
 
   * A new workspace, with a root folder named `angular-tour-of-heroes`.
-  * An initial skeleton app project, also called `angular-tour-of-heroes` (in the `src` subfolder).
+  * An initial skeleton app project, called `app` (in the `src` subfolder).
```
Comment (gkalpak) [inline aio/content/tutorial/toh-pt0.md:43]: This is not worded very clearly, but basically what this is trying to convey is this:
- A new project is created in the workspace.
- The project is named `angular-tour-of-heroes` as far as the CLI workspace config is concerned. I.e. `angular-tour-of-heroes` is used as the project name in `angular.json` and CLI commands that expect a `<project>` argument should be called with `angular-tour-of-heroes` to target this project.
- The source code of the project is inside the `src/` directory (which is a sub-directory or the workspace root (`angular-tour-of-heroes/`). (I.e. the whole `src/` directory is code for the `angular-tour-of-heroes` project, not just `src/app/`.)

I'll leave the word-smithing to the docs team, though :grin:

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Comment by aikidave** [comment]
@kapunahelewong Can you have a look at this wording?

File: aio/content/tutorial/toh-pt0.md (hunk 1 above)
Comment (aikidave) [inline aio/content/tutorial/toh-pt0.md:43]:
```suggestion
  * An initial set of files and folders that Angular requires to launch a basic application.
```

Perhaps this is sufficient? I would hate to overload the reader with a lot of details they may not find necessary.

**Comment by AndrewKushnir** [comment]
@geromegrignon thanks for creating this PR. It looks like the wording was updated in that guide recently. I'll close this PR for now. Feel free to reopen or create a new one if any additional updates are needed. Thank you.

**Pull request closed without merging**